ZippyIdSDK/ZippyIdSDK/Network/ApiClient.swift

```diff
@@ -38,11 +36,14 @@ class ApiClient {
     }
     
     func sendImages(token: String, document: Document, selfie: UIImage, documentFront: UIImage, documentBack: UIImage?, customerUid: String) -> Future<String> {
+        print("Sending images with customerUid: \(customerUid)")
```

Comment:
🚨

Reply:
Vajadzētu izvairīties no print konsolē, jo tas pastāvēs arī produkcijas buildā. Ja piemēram ir pieslēgts crashlytics, vai kāds cits crash vākšanas tool, tas jādara caur to.